components/input/input.css

```diff
@@ -0,0 +1,59 @@
+.component-input {
+    position: relative;
+    font-family: var(--primary-font);
+}
+
+.input-box {
+    border-radius: 0.5rem;
+    border: 0.15rem solid var(--primary-color);
+    background-color: transparent;
+    padding: 0.7rem;
+    outline: none;
+    font-family: inherit;
+    font-weight: bold;
+}
+.input-box-success{
+    border: 0.15rem solid var(--primary-dark-green);
+}
+.input-box-error{
+    border: 0.15rem solid var(--primary-dark-red);
+}
+
+.input-label {
+    position: absolute;
+    left: 0.8rem;
+    top: 1rem;
+    font-family: inherit;
+    font-size: 1rem;
```

Comment:
font-family will be inherited by default so you could safely remove it

Reply:
Removed! Thank you!